apps/desktop/src/components/v3/BranchList.svelte

```diff
@@ -57,6 +66,30 @@
 	const selectedBranchName = $derived(selection.current?.branchName);
 
 	let newBranchModal = $state<ReturnType<typeof NewBranchModal>>();
+
+	let conflictResolutionConfirmationModal =
+		$state<ReturnType<typeof ConflictResolutionConfirmModal>>();
+
+	async function handleUncommit(commitId: string, branchName: string) {
+		await stackService.uncommit({ projectId, stackId, branchName, commitId: commitId });
+	}
+
+	function openCommitMessageModal() {
+		// TODO: Implement openCommitMessageModal
```

Comment:
@estib-vega if you wouldn't mind, could you investigate if this needs to be fixed or removed?